apps/loan/loan-products/src/app/components/loan-products-grid/loan-products-grid.component.html

```diff
@@ -36,7 +36,11 @@ <h2 class="mb-0">
       let-expanded="expanded"
       let-rowIndex="rowIndex"
     >
-      <tr>
+      <tr
+        [ngClass]="{
```

Comment:
FYI: This is a fine technique. I prefer `[class.row-selected]="quoteSelected() === rowIndex"` so the classes don't grow into a big dynamic JSON object.